frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -370,11 +369,7 @@ public void popupMenuWillBecomeVisible(PopupMenuEvent popupMenuEvent) {
       new Timer().schedule(new TimerTask() {
         @Override
         public void run() {
-          GuiUtils.invokeLaterIfNeeded(() -> {
```

Comment:
Not that we should care much about 2020.2 at this point... but how does it affect the old IntelliJ versions?

Reply:
The selection will has a smaller width (not the size of entire pane) but this still should look OK (just like the highlighting in 2020.3)